client/webserver/site/src/html/forms.tmpl

```diff
@@ -1,3 +1,24 @@
+{{define "walletConfigTemplates"}}
+<div class="py-2 px-1 d-flex justify-content-between align-items-center">
+  Wallet Settings
+  <div data-tmpl="fileSelector" class="d-inline-block hoverbg pointer fs14"><span class="ico-textfile mr-1"></span> load from file</div>
+  <input data-tmpl="fileInput" type="file" class="form-control select d-none">
+</div>
+<div data-tmpl="dynamicOpts" class="d-flex align-items-stretch justify-content-between flex-wrap dynamicopts">
+  <div data-tmpl="textInput" class="px-0 d-flex flex-column justify-content-end">
+    <label class="pl-1 mb-1 small"> <span class="ico-info"></span></label>
+    <input type="text" class="form-control select">
+  </div>
+  <div data-tmpl="checkbox" class="pl-4 d-flex align-items-center justify-content-center">
+    <div>
+      <input class="form-check-input" type="checkbox" value="">
```

Comment:
Can we `disabled` the `input` until https://github.com/decred/dcrdex/pull/562 goes in, forcing it to be unchecked on submission?  Or would that break the form submission entirely?

Reply:
I'll just comment out the `ConfigOption` for now. I just wanted to show off the boolean/checkbox inputs, really.